build.gradle

```diff
@@ -56,9 +59,9 @@ public class InstallPath extends DefaultTask {
      * @param ouput destination directory
      */
     private void copy(String input, String output) {
-        project.copy {
-            from project.file(input)
-            into project.file(output)
+        fs.copy {
+            from input
+            into output
         }
     }
 
```

Comment:
This didn't build for me, because `install` is annotated and private, see [userguide](https://docs.gradle.org/9.1.0/userguide/validation_problems.html#private_method_must_not_be_annotated):

```diff
-    private void install() {
+    public void install() {
```